assets/blocks/quiz/question-block/question.scss

```diff
@@ -120,18 +120,23 @@ $question-input-border-color: var(--wp--preset--color--foreground, #155E65 );
 .sensei-lms-question__answer-feedback {
 	--color: #DCDCDE;
 	$horizontal-padding: 24px;
```

Comment:
I believe this is orphaned code now, so we should remove it.

Reply:
Right, removed it here https://github.com/Automattic/sensei/pull/7171/commits/696ee723976d8a1a002c0faac99223592481d751